python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py

```diff
@@ -104,7 +106,7 @@ class TableMetadata:
     retention_period_in_days: int | None
     source_dataset_name: str | None = None
     source_dataset_location: str | None = None
```

Comment:
This will need changing to the new name on line 133 too
```suggestion
    where_to_access_dataset: str | None = None

Reply:
Done